b-locker-web/src/components/organisation/topbar/orgTopbar.scss

```diff
@@ -0,0 +1,14 @@
+@import "../../../styles/org-global.scss";
+
+.topbar {
+  overflow: hidden;
+  background-color: $white;
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.19);
+
+  a {
+    float: right;
+    text-align: center;
+    padding: 10px 16px;
+    text-decoration: none;
+  }
```

Comment:
please double check if this interferes with other <a> tags. For example the ones on the main page. If this does happen, give the tags a classname and edit them that way.